src/components/new-conversation-dialog/create-room/index.js

```diff
@@ -68,7 +74,7 @@ const CreateRoom = ({
         onChange={onGroupNameChange}
         description="Should represent the topic of the group."
         maxLength={30}
-        width={260}
+        width={410}
```

Comment:
In design, it's 260.

Reply:
in the review process we said we need to make it wider. we should also fix it in the design then 👍

Reply:
Carina changed it in the design now